VAMobile/src/testUtils.tsx

```diff
@@ -204,12 +204,6 @@ function render(ui, { preloadedState, navigationProvided = false, queriesData, .
           retry: false,
         },
       },
-      logger: {
-        log: console.log,
-        warn: console.warn,
-        // Silence the error console
-        error: () => {},
-      },
```

Comment:
This is deprecated

"Custom loggers were already deprecated in 4 and have been removed in this version. Logging only had an effect in development mode, where passing a custom logger is not necessary."